index.html

```diff
@@ -17,8 +17,7 @@
         </li>
         <li>
           <a class="nav-link" href="What you should already know"
```

Comment:
- [ ] I would suggest you to kindly remove this because it doesn't have any usefulness in the project, doing this will make your project looks great and clean

```HTML
<ul>
     <li></li>
     <li></li>
     <li></li>
     <li></li>
     <li></li>
</ul>
```